src/packaging/webots_distro.c

```diff
@@ -1181,7 +1190,7 @@ static void create_file(const char *name, int m) {
       fprintf(fd, "cp /usr/lib/x86_64-linux-gnu/libfreetype.so.6 debian/usr/local/webots/lib\n");
       fprintf(fd, "cp /lib/x86_64-linux-gnu/libpci.so.3 debian/usr/local/webots/lib\n");
       fprintf(fd, "cd debian/usr/local\n");
-      fprintf(fd, "tar cjf ../../../%s-%s-%s.tar.bz2 %s\n", application_name_lowercase_and_dashes, package_version, arch2,
+      fprintf(fd, "tar cf ../../../%s-%s-%s.tar.bz2 --use-compress-prog=pbzip2 %s\n", application_name_lowercase_and_dashes, package_version, arch2,
```

Comment:
This improves the creation of the tar.bz2 package from 4:01 minutes to 1:03 minutes